src/fe/inf_fe.C

```diff
@@ -217,7 +217,7 @@ void InfFE<Dim,T_radial,T_map>::reinit(const Elem * inf_elem,
       // right now, and it will generalize a bit, and it won't break
       // the assumptions elsewhere in InfFE.
       std::vector<Point> radial_pts;
-      for (auto p : index_range(*pts))
+      for (std::size_t p=1; p < pts->size(); ++p)
```

Comment:
What happened here?  Why are you skipping p=0?

Reply:
Oh; you are right. Here I must have mixed up the commit my merging the master...